server/src/HatoholArmPluginGateHAPI2.cc

```diff
@@ -1138,7 +1167,8 @@ static bool parseItemParams(JSONParser &parser, ItemInfoList &itemInfoList,
 		PARSE_AS_MANDATORY("brief", itemInfo.brief, errObj);
 		parseTimeStamp(parser, "lastValueTime", itemInfo.lastValueTime, errObj);
 		PARSE_AS_MANDATORY("lastValue", itemInfo.lastValue, errObj);
-		PARSE_AS_MANDATORY("itemGroupName", itemInfo.itemGroupName, errObj);
+		if (!getItemGroupName(itemInfo.itemGroupName))
+			return false;
```

Comment:
For now, this line's `return false` breaks previous our err checking code.
Could you add error messages in `errObj`?